[2024-05-05 13:49:40,960] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-04-18T00:00:00+00:00 [queued]>
[2024-05-05 13:49:40,998] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-04-18T00:00:00+00:00 [queued]>
[2024-05-05 13:49:41,000] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 13:49:41,001] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-05-05 13:49:41,002] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-05-05 13:49:41,023] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): poastgres_to_s3> on 2024-04-18T00:00:00+00:00
[2024-05-05 13:49:41,030] {standard_task_runner.py:52} INFO - Started process 7848 to run task
[2024-05-05 13:49:41,038] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_prostgres_hooks_v2', 'poastgres_to_s3', '2024-04-18T00:00:00+00:00', '--job-id', '118', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_hooks.py', '--cfg-path', '/tmp/tmpl98lcqhr', '--error-file', '/tmp/tmp10tjf3re']
[2024-05-05 13:49:41,043] {standard_task_runner.py:77} INFO - Job 118: Subtask poastgres_to_s3
[2024-05-05 13:49:41,147] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_prostgres_hooks_v2.poastgres_to_s3 2024-04-18T00:00:00+00:00 [running]> on host d4a0c764ecae
[2024-05-05 13:49:41,239] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mark
AIRFLOW_CTX_DAG_ID=dag_with_prostgres_hooks_v2
AIRFLOW_CTX_TASK_ID=poastgres_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2024-04-18T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-18T00:00:00+00:00
[2024-05-05 13:49:41,262] {base.py:74} INFO - Using connection to: id: postgres_localhost. Host: host.docker.internal, Port: 5432, Schema: test, Login: airflow, Password: XXXXXXXX, extra: None
[2024-05-05 13:49:41,304] {dag_with_postgres_hooks.py:32} INFO - Saved orders data in text file: dags/get_orders_20240418.txt
[2024-05-05 13:49:41,307] {base_aws.py:362} INFO - Airflow Connection: aws_conn_id=minio_conn
[2024-05-05 13:49:41,325] {base_aws.py:164} INFO - Credentials retrieved from extra_config
[2024-05-05 13:49:41,331] {base_aws.py:81} INFO - Creating session with aws_access_key_id=ROOTUSER region_name=None
[2024-05-05 13:49:41,371] {base_aws.py:151} INFO - role_arn is None
[2024-05-05 13:49:41,553] {dag_with_postgres_hooks.py:41} INFO - Orders file /tmp/tmpra8taw9220240418 has been pushed to S3!
[2024-05-05 13:49:41,556] {python.py:118} INFO - Done. Returned value was: None
[2024-05-05 13:49:41,579] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_prostgres_hooks_v2, task_id=poastgres_to_s3, execution_date=20240418T000000, start_date=20240505T134940, end_date=20240505T134941
[2024-05-05 13:49:41,620] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-05 13:49:41,653] {local_task_job.py:146} INFO - Task exited with return code 0
